types/hooks/screens/useChatUtilsHook.types.ts

```diff
@@ -4,6 +4,7 @@ interface UseChatUtilsHook {
   isLoadingFetchingChannelDetail: boolean;
   selectedChannel: ChannelList | null;
   selectedChannelKey: number;
+  fetchChannelDetail: (channel: ChannelList) => void;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of `fetchChannelDetail` function in the interface is a good move as it provides a clear contract for what the hook should implement. However, it would be beneficial to add a comment describing what this function does and what parameters it expects.

```typescript
// Fetches details for a given channel
fetchChannelDetail: (channel: ChannelList) => void;
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->